packages/web-frontend/public/index.html

```diff
@@ -98,7 +98,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <link
       rel="stylesheet"
-      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,600,700&display=swap"
```

Comment:
Is this used somewhere? I can't seem to find where.

Reply:
This one is the font used throughout, and before the titles, when set to be
`bold` were fuzzy because we hadn't imported the correct font sizes

Reply:
Oh true, I just saw the addition of the 700 initially which I haven't seen applied anywhere but I hadn't seen the 600. Great pick up. I'm still not sure if we need the 700 but no harm in including it.